docs/release/cut-sw-release.md

```diff
@@ -157,7 +157,21 @@ cd release-tools
 ./1-client-tarballs $NON_UPCOMING_VERSIONS
 ```
 
-You should get up to 8 tarballs for each version (excluding upcoming), 25-55 megs each and they should all have the version number in the name.
+You should get 7 tarballs.
+They should all have the version number in the name.
+
+-   OSG 3.3.x
+    -   32-bit (i386)
+        - osg-wn-client, el6: 23 megabytes
+    -   64-bit
+        - osg-wn-client, el6: 23 megabytes
+        - osg-wn-client, el7: 31 megabytes
+-   OSG 3.4.x
+    -   64-bit
+        - osg-afs-client, el6: 23 megabytes
+        - osg-afs-client, el7: 31 megabytes
+        - osg-wn-client, el6: 23 megabytes
+        - osg-wn-client, el7: 31 megabytes
```

Comment:
Do we need to be so specific about the tarball sizes? It's also not clear what the user of the doc should be doing with this information. If there's a size delta, is the release broken? How big of a size delta is worth bringing to the attention of the release manager? Can't this whole section be checked programmatically?

Reply:
I'll rework the test script to check the size programatically. Then we can drop this change.

Reply:
I pushed the reworked script and I tested functionality of the script.